app/views/layouts/application.html.erb

```diff
@@ -49,7 +46,7 @@
     App.emojis = <%== MultiJson.dump(Emoji.all.map(&:name)) %>;
     <% end %>
     <% cache User.all_cache_key do %>
-    App.users = <%== MultiJson.dump(ActiveModel::ArraySerializer.new(all_other_users, each_serializer: PublicUserSerializer, root: false), pretty: true) %>;
+    App.users = <%== MultiJson.dump(ActiveModel::ArraySerializer.new(User.all, each_serializer: PublicUserSerializer, root: false), pretty: true) %>;
```

Comment:
This was going to cause an issue as it’s cached across all users but would exclude the first user to load the page, will just keep it loading all users for now, not too big a deal that you could @mention yourself, but could remove current user in JS if that becomes a problem.